sheets/management/utils_test.py

```diff
@@ -0,0 +1,119 @@
+"""Tests for sheets.management.utils"""
+from unittest.mock import MagicMock
+
+import copy
+import pytest
+from django.core.management import CommandError
+from sheets.management import utils
+
+
+@pytest.fixture(name="valid_enum_rows")
+def fixture_valid_enum_rows():
+    """Mock rows response for tests"""
+    return [
+        (
+            1,
+            [
+                "fake-purchase-order",
+                "fake-coupon-name",
+                "10",
+                "fake-text-id",
+                "fake-company",
+            ],
+        ),
+        (
+            2,
+            [
+                "fake-purchase-order-2",
+                "fake-coupon-name-2",
+                "20",
+                "fake-text-id-2",
+                "fake-company-2",
+            ],
+        ),
+    ]
+
+
+@pytest.fixture(name="patched_get_enum_rows", autouse=True)
+def fixture_patched_get_enum_rows(mocker, valid_enum_rows):
```

Comment:
Nit: You can set this function's name as `patched_get_enum_rows` and get rid of `name="patched_get_enum_rows"` in the arguments above and it will have the same effect. Same goes for `valid_enum_rows` above

Reply:
This was just to silence Pylint acting up with fixtures. The documentation on Pytest says if you use the fixture in the same module then Pylint will start spitting out "redefining name from outer scope" warnings:
https://stackoverflow.com/questions/46089480/pytest-fixtures-redefining-name-from-outer-scope-pylint